hydrocron/api/controllers/timeseries.py

```diff
@@ -51,23 +53,24 @@ def gettimeseries_get(feature, feature_id, start_time, end_time, output, fields)
 
 
 def format_json(feature_lower, results: Generator, feature_id, fields):  # noqa: E501 # pylint: disable=W0613,R0912
-    """
+    """ Format the results to the file format that the user selects (geojson)
 
-    Parameters
-    ----------
-    feature_lower
-    results
-    swot_id
-    fields
-
-    Returns
-    -------
+    :param feature_lower: Lowercase version of the type of feature
+    :type feature_lower: str
+    :param results: We pass the result of the query
+    :type results: dict
```

Comment:
type doesn't match the type in the function definition; is `results` a `Generator` or a `dict`?

Reply:
my mistake. fixed.

Reply:
Is it actually a `Generator`? You're accessing it a few lines down like a dict: `results = results['Items']`

Reply:
The return type in `DynamoDataRepository` looks incorrect. According to https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/table/query.html the return type of the `query` function is dict.

Reply:
so should we change this line and remove the Generator part? @torimcd 
def get_reach_series_by_feature_id(self, feature_id: str, start_time: str, end_time: str) -> Generator:

Reply:
Yes, just fix the return type to be a dict

Reply:
agreed!